src/qcodes_contrib_drivers/drivers/Tektronix/FCA3100.py

```diff
@@ -86,8 +79,8 @@ def get_raw(self) -> np.ndarray:
         """
         assert isinstance(self.instrument, FCA3100)
         self.instrument.write('CALCulate:AVERage:STATe 0')
-        self.instrument.write('ARM:COUN {}'.format(self.instrument.samples_number.get_latest()))
-        data_str=self.instrument.ask("READ:ARRay? {}".format(self.instrument.samples_number.get_latest()))
+        self._instrument.write('ARM:COUN {}'.format(self._instrument.samples_number.get_latest()))
```

Comment:
Why this change (from `instrument` to `_instrument`)

Reply:
No reason, fixed it.